Description: disable valgrind tests
 valgrind is not supported on certain architectures. This patch disables the valgrind
 tests on these architectures so that a build can succeed nonetheless.
 .
 This is still needed because although valgrind exists on these platforms, it still
 fails execution.
Author: Simon Elsbrock
Forwarded: https://github.com/stedolan/jq/pull/196
Last-Update: 2014-06-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -42,13 +42,6 @@
 fi
 
 
-dnl Check for valgrind
-AC_CHECK_PROGS(valgrind_cmd, valgrind)
-if test "x$valgrind_cmd" = "x" ; then
-    AC_MSG_WARN([valgrind is required to test jq.])
-fi
-
-
 dnl Don't attempt to build docs if there's no Ruby lying around
 AC_ARG_ENABLE([docs],
    AC_HELP_STRING([--disable-docs], [don't build docs]))
